.github/actions/cache_setup/action.yml

```diff
@@ -84,5 +84,5 @@ runs:
         path: |
           /tmp/hermes/download/
           /tmp/hermes/hermes/
-        key: v1-hermes-${{ inputs.hermes-version }}-${{ github.run_number }}
+        key: v1-hermes-${{ inputs.hermes-version }}
```

Comment:
cherry picked 9c6c63781669bd1d6451c880671a151060220020 for stabilize prepare-hermes-workspace